memcached.c

```diff
@@ -255,7 +255,9 @@ static void stats_reset(void) {
 
 static void settings_init(void) {
     settings.use_cas = true;
+#ifndef DISABLE_UNIX_SOCKET
```

Comment:
In hopes of getting more of these windows things upstreamed, lets find out how to minimize the ifdef soup that's going to happen.

Here, I'd suggest leaving the struct's unchanged